cmd/estuary-shuttle/rpc.go

```diff
@@ -358,7 +368,7 @@ func (s *Shuttle) handleRpcAggregateStagedContent(ctx context.Context, aggregate
 			return err
 		}
 
-		if !cont.Active || cont.Failed {
```

Comment:
if the content is failed the blocks will not be on this node, so you wont be able to aggregate it, that is why the failed check was there in the first place